pkg/app/accounts/basic.go

```diff
@@ -35,6 +45,58 @@ func (m *HTTPBasicAccountManager) UserFromRequest(r *http.Request) (User, error)
 	return userFromRequest(r)
 }
 
+func (m *HTTPBasicAccountManager) UserLogin(w http.ResponseWriter, r *http.Request) error {
+	var loginTemplate = template.Must(template.New("login").Parse(`
+<!DOCTYPE html>
```

Comment:
I prefer moving HTML code into a distinguished file like `basic_login_page.html` in perspective of managing code. HDYT?

Reply:
I agree that your suggestion will be a better way to maintain source code. However, the reason why I did not create a standalone html template file is to simplify the preparation of running `./cloud_orchestrator`. Because users would also need to put `basic_login_page.html` at their CO directory if we do something like `template.ParseFiles("basic_login_template.html")`.

Based on this, do you think it's still worth to move the login page template out of `HTTPBasicAccountManager`?

Reply:
First, noting that CO already depends on its path, because of `web/intercept/js/server_connector.js`. So `basic_login_page.html` wouldn't be the only one which users need to prepare.

OTOH it's true that making a standalone HTML template file would increase dependency on path of github repository. I understood your concern, and sounds reasonable.

Reply:
> noting that CO already depends on its path, because of web/intercept/js/server_connector.js

I see, thanks! Since my goal is to prevent from having implicit file path dependencies, I think we have two proposals here:
1. Keep the AS-IS implementation, which defines `loginTemplate` as a variable of HTTPBasicAccountManager.
2. Create a standalone `basic_login_page.html` file, and set the path of this template file at `conf.toml`.

Which one would you prefer?

Reply:
I think keeping current design would be slightly better after hearing your opinion, but I don't have any strong preference. Thanks for the explanation!